docs/LogoSupport.md

```diff
@@ -0,0 +1,63 @@
+<h1>Partial Logo Support</h1>
```

Comment:
Hey thanks for sending a PR. I think we can just specify we support all icons available in simple icons.
We might not want to maintain a version of all available components?
https://simpleicons.org/?q=AdobePremierePro